core/src/main/java/com/oath/oak/OakDetachedBuffer.java

```diff
@@ -1,5 +1,14 @@
 package com.oath.oak;
 
+/**
+ * This detached buffer allows reuse of the same object and is used in:
```

Comment:
This is public interface, therefore it most important to state where it can be used and how. As return from zc get/iterator and that in can be stored and accessed later and that it may return exception if something was deleted. This is the main difference from OakReadBuffer. It is the same interface but absolutely different behavior.

Reply:
Do you suggest pasting here the detached buffers' description from the readme?